pkg/compare/parsing.go

```diff
@@ -50,6 +97,22 @@ type ReferenceTemplate struct {
 	Config ReferenceTemplateConfig `json:"config,omitempty"`
 }
 
+func (rf ReferenceTemplate) FeildsToOmit(fieldsToOmit FieldsToOmit) []*Path {
```

Comment:
i think that this function should be called additionally or at least only the validation part before running the diff. 
if there is a a ref not found in the  fields to omit its a problem in the reference, that always will occur   and no need for the user no see it on each run. I think its good to enforce ref creators to create a valid ref. so users will not get unwanted warnings that can be confusing. 

i think maybe a validation function for the ref can be a good place to add it, and call the ReferenceTemplate.validate in the complete function (with all the other validation)  or maybe  put it in the getReference func